knowledge-base/capturing-git-traffic.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Capturing Git Traffic
+description: "Learn how to capture Git HTTPS traffic while using the Fiddler Everywhere web-debugging tool."
```

Comment:
the Fiddler Everywhere web-debugging tool could be just `using Fiddler Everywhere` or `Fiddler Everywhere app`

Reply:
I am using different terms in the metadata title to further expose the Fiddler Everywhere brand to search engines (SEO)